package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "react-native-camera",
   "description": "A Camera component for React Native. Also reads barcodes.",
-  "version": "3.17.0-patch-3",
+  "version": "3.17.0-patch-5",
```

前回の #4 では #3 の PR 番号に合わせたつもりでしたので、今回からこれを踏襲すればいいかと思います。

```suggestion
  "version": "3.17.0-patch-6",
```